reporter/src/main/kotlin/reporters/AmazonOssAttributionBuilderReporter.kt

```diff
@@ -127,47 +129,35 @@ class AmazonOssAttributionBuilderReporter : Reporter {
                     usage
                 )
 
-                val attachPackageResponse = service.attachPackage(
-                    newProjectBody.projectId,
-                    attachPackage,
-                    credentials
-                ).execute()
-
-                val attachPackageBody = getBodyOrHandleError(
-                    attachPackageResponse,
+                val attachPackageBody = callService({
```

Comment:
The formatting of these calls is inconsistent, in the other calls below you put the `{` on a new line. I'd suggest to use this formatting everywhere:
```
... callService(
    { ... },
    { ... }
)
```

And for lambdas that do not fit in one line use:
```
... callService(
    { 
        ... 
    },
    { ... }
)
```

I couldn't find a recommendation for this in the Kotlin coding conventions, but this should be aligned with their rules for function signatures and I think also with most other long function calls in our code.